style.css

```diff
@@ -0,0 +1,139 @@
+#top-nav {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 50px;
+}
+
+#top-nav a {
+  text-decoration: none;
+  font-family: Poppins, sans-serif;
+  font-weight: bold;
+  font-size: 30px;
+  color: #6070ff;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  margin-left: 24px;
+}
+
+#top-nav a:hover {
+  text-decoration: underline;
+  text-decoration-color: #23a3fa;
+  text-decoration-thickness: 2px;
+}
+
+#top-nav nav {
+  width: 45px;
+  height: 30px;
+  display: flex;
+  align-items: center;
+}
+
+#home_content {
+  background-image:
+    url("img/Ellipse 11.png"),
+    url("img/Rectangle 6.png"),
+    url("img/Rectangle\ 4.png"),
+    url("img/Ellipse 3.png"),
+    url("img/Ellipse\ 5.png"),
+    url("img/Rectangle\ 7.png"),
+    url("img/Ellipse 12.png"),
+    url("img/Ellipse 4.png"),
+    url("img/Rectangle\ 4.png"),
+    url("img/Ellipse\ 13.png"),
+    url("img/Ellipse 5.png");
+  background-position:
+    19px 90px,
+    9px 15px,
+    214px 110px,
+    323px 102px,
+    125px 106px,
+    127px 98px,
+    370px 385px,
+    290px 509px,
+    240px 507px,
+    357px 392px,
+    106px 497px;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
```

Comment:
- [ ] Kindly make sure you have a white background as per the Figma design :recycle: 

- Your design:
![Screenshot from 2023-03-24 12-56-18](https://user-images.githubusercontent.com/97834160/227506313-f011385e-7095-46b3-9011-d878d7ad507a.png)

- Figma:
![Screenshot from 2023-03-24 13-12-49](https://user-images.githubusercontent.com/97834160/227506591-f72661ff-46e5-42c5-a777-c740b238a166.png)

Reply:
Accpted and corrected